src/app/pages/sharing/nfs/nfs-list/nfs-list.component.ts

```diff
@@ -15,92 +22,179 @@ import { WebSocketService } from 'app/services/ws.service';
 
 @UntilDestroy()
 @Component({
-  template: '<ix-entity-table [title]="title" [conf]="this"></ix-entity-table>',
+  templateUrl: './nfs-list.component.html',
+  styleUrls: ['./nfs-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class NfsListComponent implements EntityTableConfig<NfsShare> {
-  title = this.translate.instant('NFS');
-  queryCall = 'sharing.nfs.query' as const;
-  updateCall = 'sharing.nfs.update' as const;
-  wsDelete = 'sharing.nfs.delete' as const;
-  routeAdd: string[] = ['sharing', 'nfs', 'add'];
-  routeAddTooltip = this.translate.instant('Add Unix (NFS) Share');
-  routeEdit: string[] = ['sharing', 'nfs', 'edit'];
-  protected routeDelete: string[] = ['sharing', 'nfs', 'delete'];
-  entityList: EntityTableComponent<NfsShare>;
-  emptyTableConfigMessages = {
-    first_use: {
-      title: this.translate.instant('No NFS Shares have been configured yet'),
-      message: this.translate.instant('It seems you haven\'t setup any NFS Shares yet. Please click the button below to add an NFS Share.'),
-    },
-    no_page_data: {
-      title: this.translate.instant('No NFS Shares have been configured yet'),
-      message: this.translate.instant('The system could not retrieve any NFS Shares from the database. Please click the button below to add an NFS Share.'),
-    },
-    buttonText: this.translate.instant('Add NFS Share'),
-  };
+export class NfsListComponent implements OnInit {
+  filterString = '';
+  dataProvider = new ArrayDataProvider<NfsShare>();
```

Comment:
Take a look at AsyncDataProvider. You should be able to simplify some code related to `emptyType$`.